### ISSUE TITLE
Maps

### ISSUE THREAD

**Issue by SKSaki** [Siile/Ninslash#47]
Can we add custom maps to servers?
How?
Also dont kill me for asking so many questions
@pelya @siile

**Comment by pelya** [comment]
Yes, make your own maps using the built-in map editor, and copy them to
'maps' directory on the server. Then add them to voting menu in .cfg file.

Siile, can we use TeeWorlds maps in Ninslash?

**Comment by SKSaki** [comment]
I use Linux @pelya on a vps

**Comment by Siile** [comment]
Entity order is different from teeworlds, so just open teeworlds' map in ninslash map editor and make sure entities are as they should.

**Comment by SKSaki** [comment]
How can i edit in a Ubuntu vps tho...
I have no PC,i ssh with my android :)

**Comment by pelya** [comment]
Uhh, that's terrible.
If you have a mouse, which you can plug into your phone using OTG adapter,
and if your phone will process right and middle mouse clicks correctly
(Samsung phones use middle click as a Home button), I can add map editor
back to Android version. But I somehow need to hide it from
touchscreen-only users.

**Comment by SKSaki** [comment]
Don't hide IT.
Just make it give an warning

**Comment by pelya** [comment]
Done, map editor added back in version 0.3.0.30. It's that tiny button in
the upper-left corner, and yes, you will need a mouse just to click it.
It's unusable on my Samsung phone, because stupid Samsung remapped middle
mouse button, so I cannot even tell you if it's usable or not on Android.
